src/react-apps/applications/shared/src/components/AltinnRadioGroup.tsx

```diff
@@ -0,0 +1,58 @@
+import { RadioGroup, Typography } from '@material-ui/core';
+import { createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+
+export interface IAltinnRadioGroupComponentProvidedProps extends WithStyles<typeof styles> {
+  id?: any;
+  value: any;
+  onChange?: any;
+  className?: any;
+  name?: string;
+  row?: boolean;
+  description?: string;
+}
+
+export interface IAltinnRadioGroupComponentState {
+}
+
+const styles = () => createStyles({
+  altinnRadioGroup: {
+    marginTop: '0rem',
+    marginBottom: '0rem',
+  },
+  altinnRadioGroupWrapper: {
+    marginTop: '2.4rem',
+  },
+  altinnRadioGroupDescription: {
+    fontSize: '1.6rem',
+  },
+});
+
+class AltinnRadioGroup extends
+  React.Component<IAltinnRadioGroupComponentProvidedProps, IAltinnRadioGroupComponentState> {
+
+  public render() {
+    return (
+      <div className={this.props.classes.altinnRadioGroupWrapper}>
+        {this.props.description &&
+          <Typography classes={{ root: this.props.classes.altinnRadioGroupDescription }}>
+            {this.props.description}
+          </Typography>
+        }
+        <RadioGroup
+          classes={{ root: this.props.classes.altinnRadioGroup }}
+          onChange={this.props.onChange}
+          value={this.props.value}
+          className={this.props.className}
+          name={this.props.name}
+          id={this.props.id}
+          row={this.props.row}
```

Comment:
What is the purpose of row if it's only set to hard coded true? (ref editModalContent)

Reply:
Just to make the shared component generic. I guess you could set row to true by default if no prop is supplied.